src/Platforms/Gcc/UtestPlatform.cpp

```diff
@@ -76,29 +76,33 @@ static int PlatformSpecificWaitPidImplementation(int, int*, int)
 
 static void GccPlatformSpecificRunTestInASeperateProcess(UtestShell* shell, TestPlugin* plugin, TestResult* result)
 {
-    pid_t cpid, w;
+    const pid_t syscallError = -1;
+    pid_t cpid;
+    pid_t w;
     int status;
 
     cpid = PlatformSpecificFork();
 
-    if (cpid == -1) {
+    if (cpid == syscallError) {
```

Comment:
Could you try not to make changes that do not change anything and are probably a matter of style? Or at least put these changes in a separate pull requests so we can discuss them there. I personally do not think the syscallError variable adds any benefit or makes it clearer. Similarly, the splitting of cpid and w in two lines makes no change.

Reply:
I added syscallError to not use the same magic number `-1` yet another time and aligned whole function to be consistent.
If I'm the only one that finds it not very trivial knowledge that -1 is the error and other value is not, I'll roll back to -1 usage as it was before me.

Reply:
Its ok. No need to roll back. If the error code was 6384 then creating a variable for that makes sense. However, -1 is such a common error code number and it is so unlikely to change that creating a variable for that is mostly a matter of taste.

I'm good with making code improvements (promote it even) but they shouldn't be mostly a matter of taste. E.g. extracting the signal logic to a new function IMHO would be clearly an improvement. Still, having them in separate PR is preferred as I'll need to understand whether they are fixing the issue or whether they are just code improvements.